bitnami/grafana/10/debian-11/rootfs/opt/bitnami/scripts/libgrafana.sh

```diff
@@ -327,6 +327,11 @@ grafana_install_plugins() {
             plugin_id="${plugin_id_version_array[0]}"
             plugin_version="${plugin_id_version_array[1]}"
             info "Installing plugin ${plugin_id} @ ${plugin_version}"
+        elif grep ' ' <<< "$plugin"; then
+            read -r -a plugin_id_version_array <<< $plugin
```

Comment:
Please doublequote to prevent globbing ([SC2086](https://www.shellcheck.net/wiki/SC2086)):
```suggestion
            read -r -a plugin_id_version_array <<< "$plugin"
```